lib/notifiers/notifiers_test.go

```diff
@@ -101,9 +116,68 @@ func TestMakeCELPredicate(t *testing.T) {
 			wantMatch: true,
 		}, {
 			name:      "substitutions mismatch",
-			filter:    `build.substitutions["DINNER"] == "PIZZA"`,
+			filter:    `"DINNER" in build.substitutions && build.substitutions["DINNER"] == "PIZZA"`,
 			build:     &cbpb.Build{Substitutions: map[string]string{"DESSERT": "CANNOLI"}},
 			wantMatch: false,
+		}, {
+			name:      "before timestamp",
+			filter:    `timestamp("2020-01-01T10:00:20.021-05:00") <= build.start_time`,
+			build:     &cbpb.Build{StartTime: convertToTimestamp("2019-01-01T10:00:20.021-05:00")},
```

Comment:
So instead of 
```
convertToTimestamp("2019-01-01T10:00:20.021-05:00")
```
we could have something like
```
convertToTimestamp(t, "2019-01-01T10:00:20.021-05:00")
```